prefix_finder/frontend/templates/results.html

```diff
@@ -115,9 +115,9 @@ <h4>Sectors</h4>
                 </div>
 
                 <footer class="card__controls">
-                  <a href="{{ result.url }}" class="button" target="_blank"><i class="material-icons">link</i></a>
+                  <a href="{{ result.url }}" class="button" target="_blank"><i class="material-icons">launch</i></a>
```

Comment:
@BobHarper1 This and line 177 are missing a title. Should they have "Visit the website for this list" to match line 59?

Reply:
Thanks Ben, yes!